mungegithub/mungers/matchers/interactions.go

```diff
@@ -125,3 +125,10 @@ func MungerNotificationName(notif string) Matcher {
 		NotificationName(notif),
 	)
 }
+
+func NonAuthorHumanActor(authorName *string) Matcher {
```

Comment:
I have the feeling that this should be done directly in the calling code. The helper doesn't add much.